speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/mainmenu.cpp

```diff
@@ -76,7 +76,7 @@ onCreditsMenuActivate(void * /* dummy */)
 static void
 onExitMenuActivate(void * /*dummy*/)
 {
-    GfuiScreenActivate(ExitMenuInit(MenuHandle));
+    GfuiScreenActivate(ExitMenuInit(MenuHandle, false, 0));
```

Comment:
The third parameter is of type enum, but that is hidden from this, add a define for 'NO_EXIT' or something similar.